src/cljs_life/core.cljs

```diff
@@ -6,14 +6,16 @@
 
 (def world-size 500)
 (def size 5)
-(def max-cells (inc (/ world-size size)))
+(def max-cells (/ world-size size))
 
 (defn neighbours
   "Returns the coordinates of the cell's 8 neighbours"
   [[x y]]
-  (for [dx [-1 0 1]
+  (filter
+    (defn f [c] (every? #(and (< %1 max-cells) (>= %1 0)) c))
```

Comment:
Sorry, it's been a while since I put this little project together so I don't quite remember all the details. What are you filtering for here?